src/Compilers/CSharp/Test/Syntax/LexicalAndXml/LexicalErrorTests.cs

```diff
@@ -447,6 +447,860 @@ public static int Main()
     Diagnostic(ErrorCode.ERR_UnexpectedCharacter, "").WithArguments(@"\u0303"));
         }
 
+        [Fact]
+        public void CS9000ERR_Newlines_are_not_allowed_inside_a_non_verbatim_interpolated_string1()
```

Comment:
>CS9000ERR [](http://example.com/codeflow?start=20&length=9)

Let's do a bulk replace on `CS9000` and `CS9001` with final error code before merging

Reply:
Consider also doing a bulk replace on `Newlines_are_not_allowed_inside_a_non_verbatim_interpolated_string`

Reply:
yup yup.